portfolio/src/main/webapp/style.css

```diff
@@ -22,7 +22,6 @@ body{
   border: 1px dotted #edffff;
   overflow: auto;
 }
-
```

Comment:
I at least would prefer the `.css` file with blank lines between style definitions.